src/elements/emby-textarea/emby-textarea.js

```diff
@@ -52,16 +52,36 @@ function AutoGrow(textarea, maxLines) {
             newHeight = self.maxAllowedHeight;
         } else {
             textarea.style.overflowY = 'hidden';
-            textarea.style.height = 'auto';
+            // textarea.style.height = 'auto';
             newHeight = textarea.scrollHeight/* - offset*/;
         }
         textarea.style.height = newHeight + 'px';
     }
 
     // Call autogrowFn() when textarea's value is changed
-    textarea.addEventListener('input', autogrowFn);
+    textarea.addEventListener('input', function() {
+        autogrowFn();
+        if (textarea.value.trim() === '') {
+            textarea.style.height = 'auto';
+        }
+    });
     textarea.addEventListener('focus', autogrowFn);
```

Comment:
Currently, the text area only grows and trimmed only when clearing.
Maybe it is better to trim on `blur`?
```suggestion
    textarea.addEventListener('input', autogrowFn);
    textarea.addEventListener('focus', autogrowFn);
    textarea.addEventListener('blur', () => {
        textarea.style.height = 'auto';
        autogrowFn();
    });
```

`focus` here is redundant, imo. The text area should already be sized correctly. :man_shrugging:

Reply:
Yea. Got it